src/code/models/node.ts

```diff
@@ -361,7 +361,8 @@ export class Node extends GraphPrimitive {
         allowNegativeValues: this.allowNegativeValues,
         valueDefinedSemiQuantitatively: this.valueDefinedSemiQuantitatively,
         frames: _.clone(this.frames),
-        combineMethod: this.combineMethod
+        combineMethod: this.combineMethod,
+        image: this.image
```

Comment:
I am wondering why we didn't an `image` key in the export format before this.

I guess I also thought that the node was supposed to get its image from the `palettItem`.  But that might be ancient history now.